mne/io/nedf/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""NEDF file import module"""
+
+# Author: Tristan Stenner <nedf@nicht.dienstli.ch>
```

Comment:
😆

Reply:
I wasn't sure if I should put my work email address in there.

Reply:
I like the domain! You can decide yourself which email address you want to share.